src/geo_types_from_wkt.rs

```diff
@@ -995,8 +995,8 @@ mod tests {
         let err = not_a_collection.unwrap_err();
         match err {
             Error::MismatchedGeometry {
-                expected: "geo_types::geometry_collection::GeometryCollection<f64>",
-                found: "geo_types::point::Point<f64>",
+                expected: "geo_types::geometry::geometry_collection::GeometryCollection",
```

Comment:
From https://doc.rust-lang.org/std/any/fn.type_name.html:

> This is intended for diagnostic use. The exact contents and format of the string returned are not specified, other than being a best-effort description of the type.

In this case, `geo_types::geometry` is just a test module, but the compiler happens to pick it since 1.65. This works on my machine, but I think these tests are pretty broken.

Reply:
Do you have a suggestion for making it less broken?

Reply:
Not sure off the top of my head. Maybe we can drop the type name from that error and include an enum?

Reply:
When you say "these tests are pretty broken" do you mean only that they are brittle? Or do you mean something else?

Reply:
Yes, sorry, brittle. We could ask if type_name is at least stable on a given toolchain version.

Reply:
I asked this on https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/.E2.9C.94.20How.20stable.20is.20the.20output.20of.20.60std.3A.3Aany.3A.3Atype_name.60.3F.

Reply:
For the record, I'd guess that this test failed because of changes made to geo_types in the meanwhile, rather than any change in the rust compiler toolchain.

geo-types now has:
1. a default trait impl `=f64`
2. moved the geometry_collection (private) module into the geometry module 

If you wanted to make this test more robust, you could check for the presence of a substring or something rather than an exact string match, or just delete the test.

Reply:
Ah, I didn't even notice`geo-types` in there. Yes, it makes sense, then. Thanks for following up.